dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/LinkageChecker.java

```diff
@@ -110,6 +111,20 @@ public static LinkageChecker create(Bom bom, Path exclusionFile)
         classPathBuilder.resolve(managedDependencies, true, DependencyMediation.MAVEN);
     ImmutableList<ClassPathEntry> classpath = classPathResult.getClassPath();
 
+    ImmutableList<UnresolvableArtifactProblem> artifactProblems =
+        classPathResult.getArtifactProblems();
```

Comment:
The artifact problems were not not reported. For Cloud SQL's case, it was unclear why MaximumLinkageErrorsTest 
 fails https://github.com/GoogleCloudPlatform/cloud-sql-jdbc-socket-factory/issues/507#issue-901373997 until I ran it with the debugger.